class.jetpack-network.php

```diff
@@ -449,12 +455,13 @@ public function do_subsiteregister( $site_id = null ) {
 				'gmt_offset'            => $gmt_offset,
 				'timezone_string'       => (string) get_option( 'timezone_string' ),
 				'site_name'             => (string) get_option( 'blogname' ),
-				'secret_1'              => $secrets[0],
-				'secret_2'              => $secrets[1],
+				'secret_1'              => $secret_1,
+				'secret_2'              => $secret_2,
 				'site_lang'             => get_locale(),
 				'timeout'               => $timeout,
 				'stats_id'              => $stat_id, // Is this still required?
-				'user_id'               => get_current_user_id(),
+				'user_id'               => $user_id,
+				'state'                 => $user_id
```

Comment:
can you add a quick comment about why we have to send the $user_id in two separate keys?